redis-om-spring/src/test/resources/application.yaml

```diff
@@ -0,0 +1,4 @@
+redis:
+  om:
+    spring:
+      maxQueryLimit: 0
```

Comment:
Can we namespace this more? Thinking of the other properties we'll need, maybe something like:

```
redis:
  om:
    spring:
       repository:
          query:
              limit: xxx
              timeout: xxx
              language:  ...
```